samples/android/app/src/main/AndroidManifest.xml

```diff
@@ -15,6 +15,7 @@
         android:icon="@mipmap/ic_launcher"
         android:label="@string/app_name"
         android:supportsRtl="true"
+        android:usesCleartextTraffic="true"
```

Comment:
Is this just for tests or do we need to support cleartext traffic in normal operation?  If it's just for tests can we put this in a separate `AndroidManifest.xml` in the `androidTest` directory so as to not relax security on normal operation?

Reply:
I did not know that we can separate this out.
I'm not even sure why we need it, it worked before?
But maybe I updated the SDK used in my emulator and thus non-cleartext is now enforced.
Would be good if someone can run that test on their machine to see its effect.

Reply:
Yes, this is only required for `androidTest`, [see my comment here](https://github.com/mozilla-mobile/android-components/blob/5ec752f2587432e369b905f6de68cebabfd63ada/samples/glean/src/main/AndroidManifest.xml#L13-L15).